velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -310,7 +302,8 @@ void MinMaxByAggregationTestBase::SetUp() {
   AggregationTestBase::SetUp();
   AggregationTestBase::disallowInputShuffle();
 
-  for (const TypeKind type : kSupportedTypes) {
+  for (const TypePtr typePtr : kSupportedTypes) {
+    auto type = typePtr->kind();
     switch (type) {
```

Comment:
I did this to avoid changing type to kind in `CASE` branches:
```
 case TypeKind::BOOLEAN:
        dataVectorsByType_.emplace(type, buildDataVector<bool>(numValues_));
        break;
      case TypeKind::TINYINT:
        dataVectorsByType_.emplace(type, buildDataVector<int8_t>(numValues_));
        break;
```